rclpy/services/minimal_client/examples_rclpy_minimal_client/client.py

```diff
@@ -31,12 +31,14 @@ def main(args=None):
     future = cli.call_async(req)
     rclpy.spin_until_future_complete(node, future)
 
-    if future.result() is not None:
+    try:
+        result = future.result()
+    except Exception as e:
+        node.get_logger().info('Service call failed %r' % (e,))
+    else:
```

Comment:
nit: instead of a separate else-block, I would have preferred to move the successful log statement in try-block.

Reply:
I'd probably write it the way you mentioned in practice. If it seems reasonable to you, as an example I like that the try/except block is around the minimum amount of code. Maybe it would help someone reading the example understand what's expected to raise?

Reply:
Sounds fine to me.